### PR TITLE
rubocop installed

### PR DESCRIPTION
**Pull request by FayeCarter**

### CODE REVIEW

File: .rubocop-https---raw-githubusercontent-com-makersacademy-scaffolint-v1-1-0--rubocop-yml [hunk 1]
```diff
@@ -0,0 +1,489 @@
+#    _____               ____ ____        __     _         __
+#   / ___/ _____ ____ _ / __// __/____   / /    (_)____   / /_
+#   \__ \ / ___// __ `// /_ / /_ / __ \ / /    / // __ \ / __/
+#  ___/ // /__ / /_/ // __// __// /_/ // /___ / // / / // /_
+# /____/ \___/ \__,_//_/  /_/ 1 \____//_____//_//_/ /_/ \__/
+#
```
Comment (Edward-Phillips) [inline .rubocop-https---raw-githubusercontent-com-makersacademy-scaffolint-v1-1-0--rubocop-yml:6]: love me some asciii art

**Review by Edward-Phillips — APPROVED** [review]
LGTM

File: Gemfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
+gem 'rubocop', '0.79.0'
```
Comment (hturnbull93) [inline Gemfile:8]: F.A.B. :+1:

**Review by hturnbull93 — APPROVED** [review]
LGTM :+1:

**Pull request merged**